Model/Consumer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+declare(strict_types=1);
+
+namespace Ziffity\Task\Model;
+
+use Magento\Framework\Exception\LocalizedException;
+use Ziffity\Task\Model\Repository;
+
+
+class Consumer
+{
+    
+    private $logger;
+    
+    private $customCartRepository;
+
+    public function __construct(
+        \Psr\Log\LoggerInterface $logger,
```

Comment:
@NandhiniKarthi If you use FQDN you need to be consistent. Pls don't mix and match. Declare above the class.